# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/serviceworkers/moz.build
# Commit: afae3c50f7ac
# Full Hash: afae3c50f7acff60d9400adbdc3a8aa2f9d7bdfe
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2021-06-10 15:47:45
# Regressor Bug: 1564235
# File Overlap Count: 3
# Description:
#   Bug 1564235 - P2 Create NavigationPreloadManager.webidl and implement webidl binding. r=dom-worker-reviewers,webidl,asuth
#   
#   This patch creates NavigationPreloadManager.webidl and implements the webidl binding.
#   
#   Depends on D116913
# ==============================================================================

diff -r 2148ef30186a -r afae3c50f7ac dom/serviceworkers/moz.build
--- a/dom/serviceworkers/moz.build	Thu Jun 10 06:01:26 2021 +0000
+++ b/dom/serviceworkers/moz.build	Thu Jun 10 06:01:26 2021 +0000
@@ -13,6 +13,7 @@
     "FetchEventOpParent.h",
     "FetchEventOpProxyChild.h",
     "FetchEventOpProxyParent.h",
+    "NavigationPreloadManager.h",
     "ServiceWorker.h",
     "ServiceWorkerActors.h",
     "ServiceWorkerChild.h",
@@ -46,6 +47,7 @@
     "FetchEventOpParent.cpp",
     "FetchEventOpProxyChild.cpp",
     "FetchEventOpProxyParent.cpp",
+    "NavigationPreloadManager.cpp",
     "RemoteServiceWorkerContainerImpl.cpp",
     "RemoteServiceWorkerImpl.cpp",
     "RemoteServiceWorkerRegistrationImpl.cpp",